stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Android.kt

```diff
@@ -20,4 +23,17 @@ class Android {
             viewModel.bindView(channelListHeaderView, viewLifecycleOwner)
         }
     }
+
+    /**
+     * @see <a href="https://getstream.io/chat/docs/android/messages_header_view">Messages Header View</a>
+     */
+    inner class MessageListHeader(private val messagesHeaderView: MessagesHeaderView) : Fragment() {
```

Comment:
Why is this class `inner`? Also, fragment with constructor args looks weird.

Reply:
They can be not inner and private too... But please mind that this is just a code snipet that only has to compile correctly, so we know that no braking changes were made in those classes.

Reply:
Better `public` as this is a documentation... changed to `public`.